config/environments/development.rb

```diff
@@ -21,6 +21,16 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_host_name: 's3-us-west-1.amazonaws.com',
+    s3_credentials: {
+      bucket: ENV['S3_BUCKET_NAME'],
+      access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+      secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+  }
+}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.